From db8be6458fb97cc3ccba9a576e21b266d28475ca Mon Sep 17 00:00:00 2001
From: Tristan Bastian <tristan-c.bastian@gmx.de>
Date: Wed, 16 Jan 2019 19:09:55 +0100
Subject: [PATCH 2/2] nyan-big: add padding to its file

Without this padding nyan-big ends at a blank screen on boot.
Details on how to get to this padding can be found in the README.chromium under Notes.

Signed-off-by: Tristan Bastian <tristan-c.bastian@gmx.de>
---
 doc/chromium/nyan-big.its | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/chromium/nyan-big.its b/doc/chromium/nyan-big.its
index 37d4e10949..bd412915e9 100644
--- a/doc/chromium/nyan-big.its
+++ b/doc/chromium/nyan-big.its
@@ -6,7 +6,7 @@

        images {
                kernel {
-                       description = "U-Boot mainline";
+                       description = "U-Boot mainline    ";
                        type = "kernel_noload";
                        arch = "arm";
                        os = "linux";
-- 
2.19.1

